src/black_primer/cli.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python3
-
 # coding=utf8
```

Comment:
Maybe remove this comment too? As far as I know utf8 encoding is the default in Python 3.

Reply:
Not on windows.